src/main/java/cofh/thermaldynamics/util/RecipeCover.java

```diff
@@ -11,13 +11,14 @@
 import net.minecraftforge.common.ForgeHooks;
 
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 public class RecipeCover implements IRecipe {
 
 	public static final RecipeCover instance = new RecipeCover();
 
 	@Override
-	public boolean matches(@Nonnull InventoryCrafting craft, @Nonnull World world) {
```

Comment:
I just checked the IRecipe class and `world` is supposed to be `nonnull`. In this case, it is a "bug" in thermal expansion. Should I revert his change?